library/kani/src/aliasing.rs

```diff
@@ -0,0 +1,254 @@
+const STACK_DEPTH: usize = 15;
+type PointerTag = u8;
+
+use crate::mem::pointer_object;
+use crate::mem::pointer_offset;
+use crate::shadow::ShadowMem;
+use crate::{any, assume};
+
+/// The stacked borrows state.
+pub mod sstate {
```

Comment:
As a person who is not very familiar with stacked borrows, I would appreciate better module-level documentation, explaining what are those things for.